src/DependencyInjection/PrefixSeedCompilerPass.php

```diff
@@ -16,7 +16,7 @@ class PrefixSeedCompilerPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container): void
     {
-        if ($container->hasParameter('cache.prefix.seed')) {
+        if ($container->hasParameter('cache.prefix.seed')) { /** @phpstan-ignore-line */
```

Comment:
I think PHPStan may be on to something here. At some point this went from defaulting to `null` to defaulting to `%kernel.project_dir%.%kernel.container_class%` according to [these docs](https://symfony.com/doc/current/reference/configuration/framework.html#prefix-seed). I think we can rely on this being set even if we don't set it now.

Reply:
handled :heavy_check_mark: